woocommerce/google-listings-and-ads
# Call Shipping Time API from front-end

### Changes proposed in this Pull Request:

Closes #132.

This PR calls shipping time API. The file structure mostly mimics shipping rate API integration. When we have more time in the future, it could use some refactoring to simplify things in shipping rate and shipping time.

### Screenshots:

![image](https://user-images.githubusercontent.com/417342/109538669-55374e80-7afb-11eb-8150-31a1c7b31390.png)

### Detailed test instructions:

1. Open https://gla1.test/wp-admin/admin.php?page=wc-admin&path=%2Fgoogle%2Fsetup-mc
2. Proceed to Step 3. 
3. Go to Shipping Time section. 
4. Select "I can estimate a flat shipping time for all my products."
3. Add, edit (including inline edit and edit in modal) and delete shipping times. 
4. Occasionally refresh the page to make sure the data values are saved and loaded correctly.
